### ISSUE TITLE
Feature Request: Support RT/NT

### ISSUE THREAD

**Issue by brimwats** [SkepticMystic/breadcrumbs#2]
A thesauri structure uses these elements:

```
Elements of a thesaurus1
A thesaurus is composed of five elements:

Descriptors (valid terms);
Non-descriptors (non-valid terms);
Hierarchical relationships;
Associative relationships;
Scope notes and definitions.
These elements are abbreviated as follows:

  U			Use
  UF			Used for
  BT			Broader term
  NT			Narrower term
  RT			Related term
  SN			Scope note
  DF			Definition
```
(example from)

Currently breadcrumbs supports BT only

it might be useful to follow the code here: https://github.com/jlhardes/metadataBias

results in a similar matrix to: https://ihlia.nl/search/index.jsp?q%3Asearch=gay&q%3Azoekterm.row1.field3=&lang=en

**Comment by SkepticMystic** [comment]
Let me know if `0.1.1` achieves this :) Also, when I fix #7, this should be even more enhanced

**Comment by brimwats** [comment]
This seems to be resolved!
